styles.css

```diff
@@ -95,6 +95,68 @@ nav {
     margin: auto;
 }
 
+
+
+.wordInput {
+    height: 70px;
+    width: 70%;
+    margin: 0 auto;
+    margin-bottom: 25px;
```

Comment:
Could this be all in one line (Margin: 0 auto 25px auto)?

Reply:
Nice one, I updated this